libcloud/compute/drivers/gce.py

```diff
@@ -3995,7 +3995,7 @@ def create_node(
             ex_on_host_maintenance=None, ex_automatic_restart=None,
             ex_preemptible=None, ex_image_family=None, ex_labels=None,
             ex_accelerator_type=None, ex_accelerator_count=None,
-            ex_disk_size=None):
+            ex_disk_size=None, **kwargs):
```

Comment:
I saw that other drivers like `ec2` and `azure` do this, so I followed their pattern.